app/views/public_pages/home.html.erb

```diff
@@ -116,17 +125,16 @@
   <div class="slab slab--secondary-cta slab--grey slab--half-padded">
     <div class="grid">
       <div class="grid__item width-three-fourths">
-        <h2 class="h3 spacing-below-0"><%= t("views.public_pages.home.features.header") %></h2>
-        <ul class="list--bulleted text--small text--semibold">
-          <li class="spacing-below-0"><%= t("views.public_pages.home.features.free") %></li>
-          <li class="spacing-below-0"><%= t("views.public_pages.home.features.chat") %></li>
-          <li class="spacing-below-0"><%= t("views.public_pages.home.features.backtaxes", least_recent_bt_year: TaxReturn.backtax_years.last, most_recent_bt_year: TaxReturn.backtax_years.first) %></li>
-          <li class="spacing-below-0"><%= t("views.public_pages.home.features.amendments") %></li>
+        <h2 class="h3 spacing-below-0"><%= t("views.public_pages.home.faq.header") %></h2>
+        <ul class="list--bulleted">
+          <li class="spacing-below-15"><%= link_to t("views.public_pages.faq.question_groups.stimulus.how_do_i_get_the_stimulus_payments.question"), faq_question_url(host: MultiTenantService.new(:gyr).host, section_key: :stimulus, question_key: "how_do_i_get_the_stimulus_payments"), class: "link--subtle" %></li>
+          <li class="spacing-below-15"><%= link_to t("views.public_pages.faq.question_groups.should_i_file_a_tax_return.what_are_the_potential_benefits_of_filing_a_tax_return.question"), faq_question_url(host: MultiTenantService.new(:gyr).host, section_key: :should_i_file_a_tax_return, question_key: "what_are_the_potential_benefits_of_filing_a_tax_return"), class: "link--subtle" %></li>
+          <li class="spacing-below-0"><%= link_to t("views.public_pages.faq.question_groups.should_i_file_a_tax_return.am_i_a_nonfiler.question"), faq_question_url(host: MultiTenantService.new(:gyr).host, section_key: :should_i_file_a_tax_return, question_key: "am_i_a_nonfiler"), class: "link--subtle" %></li>
```

Comment:
oh, i actually didnt know you could specify a host on a named route like this. that's nice and a bit cleaner than having to build it completely from `url_for`

Reply:
but ... if this link is on GYR I feel like you don't need to specify the host at all?